tools/osbuild-image-info

```diff
@@ -2919,15 +2919,15 @@ def is_iso(path):
 
 
 def analyse_iso(path):
-    with tempfile.TemporaryDirectory(dir="/var/tmp") as tmp:
-        report = None
-        subprocess.run(["mount", "-o", "loop", path, tmp], check=True)
+    with mount(path, ["loop"]) as tmp:
+        liveimg_tarball = os.path.join(tmp, "liveimg.tar.gz")
+        if not os.path.exists(liveimg_tarball):
```

Comment:
The call to `analyse_tarball` below would fail if the file doesn't right? Might as well not duplicate the check here.

Reply:
No, it returns an empty dict (report), which would also cause an error, after I changed the code to do so, but the error message would not be IMHO clear enough on the underlying problem.

Reply:
(it calls `tar .... filename` with `check=True` so we'll get an exception when `tar` fails to find the file I believe).